travel/src/main/resources/db/migration/V3__dbscheme.sql

```diff
@@ -0,0 +1,88 @@
+CREATE TABLE IF NOT EXISTS `exchange_rates` (
+    `id` BIGINT NOT NULL AUTO_INCREMENT,
+    `code` VARCHAR(20) DEFAULT NULL,
+    `amount_in_rsd` FLOAT UNSIGNED DEFAULT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `UK_code` (`code`)
+    );
+
+CREATE TABLE IF NOT EXISTS `travel_expense_forms` (
+    `id` VARCHAR(36) NOT NULL,
+
+    `departure_time` TIME NOT NULL,
+    `business_trip_end_date` DATE NOT NULL,
+    `business_trip_end_time` TIME NOT NULL,
+    `breakfasts_provided` INT(3) DEFAULT 0,
+    `lunches_provided` INT(3) DEFAULT 0,
+    `dinners_provided` INT(3) DEFAULT 0,
+    `submission_date` DATE NOT NULL,
+    PRIMARY KEY (`id`)
+    );
+
+CREATE TABLE IF NOT EXISTS `expenses` (
+    `id` VARCHAR(36) NOT NULL,
+    `claim` VARCHAR(255) NOT NULL,
+    `description` VARCHAR(255),
+    `amount` INT DEFAULT 1,
```

Comment:
does it have to be int?

Reply:
Thinking about it now, I would put it as unsigned tinyint, since there's no need for negative values. Would that be more fitting?